torch/cuda/random.py

```diff
@@ -48,20 +49,25 @@ def set_rng_state(new_state: Tensor, device: Union[int, str, torch.device] = 'cu
         device (torch.device or int, optional): The device to set the RNG state.
             Default: ``'cuda'`` (i.e., ``torch.device('cuda')``, the current CUDA device).
     """
-    new_state_copy = new_state.clone(memory_format=torch.contiguous_format)
-    if isinstance(device, str):
-        device = torch.device(device)
-    elif isinstance(device, int):
-        device = torch.device('cuda', device)
-
-    def cb():
-        idx = cast(torch.device, device).index
-        if idx is None:
-            idx = current_device()
-        default_generator = torch.cuda.default_generators[idx]
-        default_generator.set_state(new_state_copy)
-
-    _lazy_call(cb)
+    # Disable fake tensor tracing for set_rng_state as set_state with fake
```

Comment:
This doesn't sound right to have to modify random library code.
How does Dynamo handle these usually?